src/index.ts

```diff
@@ -77,17 +77,15 @@ function fetchAndLaunchMetals(context: ExtensionContext, javaHome: string) {
   }
 
   const config = workspace.getConfiguration("metals");
-  const serverVersionConfig: string = config.get<string>("serverVersion");
+  const serverVersionConfig = config.get<string>("serverVersion");
   const defaultServerVersion = config.inspect<string>("serverVersion")!
     .defaultValue!;
   const serverVersion = serverVersionConfig
     ? serverVersionConfig
     : defaultServerVersion;
 
-  const serverProperties = config.get<string[] | undefined>("serverProperties");
-  const customRepositories = config.get<string[] | undefined>(
-    "customRepositories "
-  );
+  const serverProperties = config.get<string[]>("serverProperties") ?? [];
+  const customRepositories = config.get<string[]>("customRepositories ");
```

Comment:
`get<A>` always returned `A | undefined`. With `strict` we would have prevented the bug we found the other day about `undefined` slipping through.